runelite-client/src/main/java/net/runelite/client/plugins/pestcontrol/PestControlPlugin.java

```diff
@@ -42,8 +51,19 @@
 )
 public class PestControlPlugin extends Plugin
 {
+	private static final Set<Integer> SPINNER_IDS = ImmutableSet.of(
+		NpcID.SPINNER,
+		NpcID.SPINNER_1710,
+		NpcID.SPINNER_1711,
+		NpcID.SPINNER_1712,
+		NpcID.SPINNER_1713
+	);
+
 	private final Pattern SHIELD_DROP = Pattern.compile("The ([a-z]+), [^ ]+ portal shield has dropped!", Pattern.CASE_INSENSITIVE);
 
+	@Getter(AccessLevel.PACKAGE)
+	private Set<NPC> spinners = new HashSet<>();
```

Comment:
this should be final and cleared on shutdown, also probably just arraylist is better